apisix/plugins/basic-auth.lua

```diff
@@ -39,6 +44,10 @@ local consumer_schema = {
     properties = {
         username = { type = "string" },
         password = { type = "string" },
+        hide_auth_header = {
```

Comment:
We don't need to configure it in the consumer

Reply:
Done.